### ISSUE TITLE
Enhancement: Ask ai to evaluate given move

### ISSUE THREAD

**Issue by nqeron** [nqeron/TakAnalysis#9]
Add in request line to get ai to return evaluation number for a given move at a position?

**Comment by Daenyth** [comment]
Yes, if I'm at a specific move I'd like to see what it thinks of various
moves following
